pkg/sql/parsers/goyacc/goyacc.go

```diff
@@ -880,14 +882,14 @@ func gettok() int {
 	switch c {
 	case EOF:
 		if tokflag {
-			fmt.Printf(">>> ENDFILE %v\n", lineno)
```

Comment:
goyacc.go is the compiler tool of parser. It does not generate output at run time, so it dose not need to modify it to log output to make it easier to see the error info when producing the parser.

Reply:
NOT agree.
not like version output, it needs stdout.
all log should be output in the same way, you can also use `Debugf` instead of `tokflag` to control this log-output in online-env.

Reply:
These are equivalent to compilation information. If someone compiles the parser separately and the log specifies the output path, the compilation error information may be ignored. This is not very friendly

Reply:
i may have misunderstood this package. Is goyacc.go just an tools? And no other pkg will call those funcitons in it.

Reply:
goyacc.go is just a tool.

Reply:
ok, i will revert this change. sorry.

Reply:
thanks